src/main/java/com/okta/tools/aws/settings/MultipleProfile.java

```diff
@@ -29,16 +30,16 @@
     static final String PROFILE_EXPIRY = "profile_expiry";
     static final String OKTA_SESSION = "okta_roleArn";
 
-    public com.okta.tools.aws.settings.Profile getProfile(String oktaprofile, Path profileIni) throws IOException {
+    public Optional<com.okta.tools.aws.settings.Profile> getProfile(String oktaprofile, Path profileIni) throws IOException {
```

Comment:
:art: Don't need the fully-qualified class name here. You can import it and use **Profile**.